src/matchers/toBeAfter/index.js

```diff
@@ -1,26 +1,26 @@
-import { matcherHint, printReceived } from 'jest-matcher-utils';
-
 import predicate from './predicate';
 
-const passMessage = (received, after) => () =>
-  matcherHint('.not.toBeAfter', 'received', '') +
+const passMessage = (received, after, context) => () =>
+  context.utils.matcherHint('.not.toBeAfter', 'received', '') +
   '\n\n' +
-  `Expected date to be after ${printReceived(after)} but received:\n` +
-  `  ${printReceived(received)}`;
+  `Expected date to be after ${context.utils.printReceived(after)} but received:\n` +
+  `  ${context.utils.printReceived(received)}`;
 
-const failMessage = (received, after) => () =>
-  matcherHint('.toBeAfter', 'received', '') +
+const failMessage = (received, after, context) => () =>
+  context.utils.matcherHint('.toBeAfter', 'received', '') +
   '\n\n' +
-  `Expected date to be after ${printReceived(after)} but received:\n` +
-  `  ${printReceived(received)}`;
-
-export default {
-  toBeAfter: (date, after) => {
-    const pass = predicate(date, after);
-    if (pass) {
-      return { pass: true, message: passMessage(date, after) };
-    }
+  `Expected date to be after ${context.utils.printReceived(after)} but received:\n` +
+  `  ${context.utils.printReceived(received)}`;
 
-    return { pass: false, message: failMessage(date, after) };
+function toBeAfter(date, after) {
+  const pass = predicate(date, after);
+  if (pass) {
+    return { pass: true, message: passMessage(date, after, this) };
```

Comment:
```suggestion
    return { pass: true, message: passMessage(date, after, this.utils) };
```

Reply:
This might be a nice change, would access a bit more directly.